Calculator.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+
+# function to return sum of 2 integers
+def sum_function(intA, intB):
+    return intA+intB
+
+# funtion to divide 2 integers and return as float
+def divide_function(intA, intB):
+    return intA/intB
+
+# function to divide 2 integers and return value rounded to nearest integer
+def divide_function_round(intA, intB):
+    return round(intA/intB)
+
+#### unit testing
+
+class TestFunctions(unittest.TestCase):
+
+
+    def setUp(self):
+        self.a = 8967678
+        self.b = 5675
+
```

Comment:
This was tricky: What with test: divide by zero? :)

Reply:
Added a new commit to address this issue